src/org/opensolaris/opengrok/analysis/posh/PoshAnalyzer.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2005, 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
just 2017 please

Reply:
OK, I fixed all the copyright dates in the files. I have not pushed them up yet because that is just going to generate another failed build.

Reply:
don't worry about failed builds, this is why CI is in place, to see builds fail ;)
(or fix)